scripts/iscan

```diff
@@ -11,8 +11,12 @@ export PS4='+ [${BASH_SOURCE[0]##*/}:${LINENO}${FUNCNAME[0]:+:${FUNCNAME[0]}}] '
 #
 # To pull the image from private Amazon ECR:
 #     REGISTRY="${aws_account_id}.dkr.ecr.${region}.amazonaws.com/" ./iscan /mount/point
+#
 REGISTRY=${REGISTRY-public.ecr.aws/elastio-dev/}
 
+DEFAULT_S3_BUCKET='elastio-rw-reports-test-bucket'
+S3_BUCKET=${S3_BUCKET-$DEFAULT_S3_BUCKET}
```

Comment:
`S3_BUCKET=${S3_BUCKET:-$DEFAULT_S3_BUCKET}` would have different semantics.  I want to preserve null `S3_BUCKET` values (i.e., empty strings).

[bash(1)](https://man7.org/linux/man-pages/man1/bash.1.html#EXPANSION):
```
       When not performing substring expansion, using the forms
       documented below (e.g., :-), bash tests for a parameter that is
       unset or null.  Omitting the colon results in a test only for a
       parameter that is unset.

       ${parameter:-word}
              Use Default Values.  If parameter is unset or null, the
              expansion of word is substituted.  Otherwise, the value of
              parameter is substituted.
```